mkosi

```diff
@@ -1076,16 +1076,22 @@ ensure that you have openssl program in your system.
 def install_fedora(args, workspace, run_build_script):
     if args.release == 'rawhide':
         last = sorted(FEDORA_KEYS_MAP)[-1]
-        die('Use numerical release for Fedora, not "rawhide"\n' +
-            '(rawhide was {} when this mkosi version was released)'.format(last))
+        warn('Assuming rawhide is version {} -- '.format(last) +
+             'You may specify otherwise with --release=rawhide-<version>')
+        args.releasever = last
+    elif args.release.startswith('rawhide-'):
```

Comment:
This is a bit of a hack, but it should work. We can always tweak the interface later.